capstone-frontend/src/components/ListPage.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+
+class ListPage extends Component {
+
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      gBooks: [],
+      loading: true,
+      empty: false
+    }
+  }
+
+
+  fetchBooks = async () => {
+
+    const userID = window.localStorage.getItem("userID");
+
+    console.log("Fetch Books LIST PAGE")
+
+    const bookLists = await fetch(`/api/booklist?userID=${userID}`, {
+      method: "GET",
+    }).then(resp => resp.json());
+
+    console.log("Booklist", bookLists)
+
+    const bookList = Array.from(bookLists).filter(bookList => (bookList.id === this.props.match.params.id))[0].gbookIDs;
+
+    console.log("Booklist", bookList)
+
+    if (bookList.length === 0) {
+      this.setState({loading: false, empty: true});
+      return;
+    }
+
+    const gBooks = [];
+
+    await Promise.all(bookList.map(async (gBookID) => {
+      const gBook = await fetch(`https://www.googleapis.com/books/v1/volumes/${gBookID}`).then(response => response.json())
```

Comment:
I also know Antonio is refactoring and adding functionality to his SearchServlet, once he does this it would be nice to route these calls through the backend rather than directly to the Google Books API. A nice way to separate concerns.

Reply:
Ok, I will refactor my code in another PR once his code is ready